src/shared-components/dialogModal/style.ts

```diff
@@ -68,6 +68,10 @@ export const ModalContainer = styled.div<{
     }
   }
 
+  @media screen and (max-width: ${BREAKPOINTS.sm}px) {
```

Comment:
You can use `MEDIA_QUERIES` directly as in line 75. We typically develop "mobile first" which really means the base styles we set are for all views, and then we start adding exceptions starting at wider viewport widths:

```diff
- max-height: 700px;
+ max-height: 100%

+ ${MEDIA_QUERIES.smUp} {
+  max-height: 700px;
+ }